src/deck/guides/konnect.md

```diff
@@ -0,0 +1,227 @@
+
+{:.note}
+> Prior to decK 1.12, decK provided `deck konnect` commands.
+These commands are deprecated and have been replaced with flags.
+
+`--konnect-addr`
+:  Address of the {{site.konnect_short_name}} endpoint. (Default: `"https://us.api.konghq.com"`)
+
+`--konnect-email`
+:  Email address associated with your {{site.konnect_short_name}} account.
+
+`--konnect-password`
+:  Password associated with your {{site.konnect_short_name}} account.
+This takes precedence over the `--konnect-password-file` flag.
+
+`--konnect-password-file`
+:  File containing the password to your {{site.konnect_short_name}} account.
+
+`--konnect-runtime-group-name`
+:  {{site.konnect_short_name}} runtime group name.
+
+## Authenticate with {{site.konnect_short_name}}
+
+decK looks for {{site.konnect_short_name}} credentials in the following order of precedence:
+
+1. Password set with the `--konnect-password` flag
+2. decK configuration file (default: `~/.deck.yaml`)
+3. File passed with the `--konnect-password-file` flag
+
+For example, if you have both a decK config file and a {{site.konnect_short_name}} password file, decK uses the password in the config file.
+
+### Authenticate with CLI flags
+
+There are two ways to authenticate through CLI flags: by passing the password in `--konnect-password` directly, or by saving it to a file and passing the filename to decK with `--konnect-password-file`.
+
+With `konnect-password`:
+
+```sh
+deck ping \
+  --konnect-email YOUR_EMAIL \
+  --konnect-password YOUR_PASSWORD
+```
+
+Or, save your {{site.konnect_short_name}} password to a file, then pass the file using the `--konnect-password-file` flag:
+
+```sh
+deck ping \
+  --konnect-email YOUR_EMAIL \
+  --konnect-password-file /PATH/TO/FILE
+```
+
+### Authenticate with a decK config file
+
+The default decK configuration file is `~/.deck.yaml`.
+You can store {{site.konnect_short_name}} credentials in this file to pass them more securely.
+For example:
+
+```
+konnect-email: example@email.com
+konnect-password: YOUR_PASSWORD
+```
+
+Alternatively, you can save your password in a separate file, then specify the password file instead of a password:
+
+```
+konnect-email: example@example.com
+konnect-password-file: PATH/TO/FILENAME
+```
+
+decK automatically uses the credentials in `~/.deck.yaml` in any subsequent calls:
+
+```
+deck ping
+
+Successfully Konnected as MyName (Konnect Org)!
+```
+
+## Target a {{site.konnect_short_name}} API
+
+Use `--konnect-addr` to select the API to connect to.
+
+The default API decK uses is `https://us.api.konghq.com`, which targets the `cloud.konghq.com` environment.
+If your account is in this environment, you don't need to change anything.
+
+If your account is in the `konnect.konghq.com` environment, use this flag to target the relevant API:
+
+```sh
+deck ping --konnect-addr https://konnect.konghq.com
+```
+
+## Runtime groups
+
+Target runtime groups in your state file with the `konnect_runtime_group` parameter:
```

Comment:
If this is an option you can add to your basic config file, I might add this as a bullet under step one with something like "If you want to target runtime groups....". I just wasn't sure how this related to the steps below.

Reply:
It's more about targeting custom runtime groups. By default, a file will target the runtime group `default`, and if you want to change that, you have to set it either in the file or with a flag. Even if you don't explicitly set it though, decK will add the parameter.